.gitignore

```diff
@@ -1,4 +1,7 @@
 # Default ignored files
-/shelf/
-/workspace.xml
-/.idea
+shelf/
+workspace.xml
```

Comment:
```suggestion
shelf/
workspace.xml
```
Could u please explain why did you add the resources above?

Reply:
These resources were in .gitignore by default on branch "master" 
![default ignored files](https://user-images.githubusercontent.com/72501713/97706740-38e0c200-1ac7-11eb-945b-f7f0d07db84e.png)

Reply:
I see thx